judge/models/problem.py

```diff
@@ -97,6 +97,9 @@ class Problem(models.Model):
     name = models.CharField(max_length=100, verbose_name=_('problem name'), db_index=True)
     description = models.TextField(verbose_name=_('problem body'))
     authors = models.ManyToManyField(Profile, verbose_name=_('creators'), blank=True, related_name='authored_problems')
+    curators = models.ManyToManyField(Profile, verbose_name=_('curators'), blank=True, related_name='curated_problems',
+                                     help_text=("These users will be able to edit a problem,"
+                                                 + " but not be publicly shown as an author."))
```

Comment:
Using `+` is ridiculous. Adjacent string literals automatically concatenate. Use single quotes to be consistent. Should probably apply that change to line below.